### ISSUE TITLE
Error msg with in bepo layer with flycheck layer on startup

### ISSUE THREAD

**Issue by Letheed** [syl20bnr/spacemacs#5153]
#### Description
Error message: `Error (use-package): flycheck post-:init hook: Symbol's value as variable is void: flycheck-error-list-mode-map` on startup with 0.105.10.

cc @StreakyCobra 

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.10
- Spacemacs branch: master (rev. 793b326)
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-sort-by-usage t)
 bepo better-defaults
 (c-c   :variables c-c  -enable-clang-support t)
 colors emacs-lisp git github
 (haskell :variables haskell-enable-ghci-ng-support t)
 markdown shell shell-scripts syntax-checking
 (rust :variables rust-enable-racer t)
 yaml)
```

**Comment by StreakyCobra** [comment]
@Letheed Have you updated your packages recently? I can not reproduce this, even when using the `hybrid` mode with a stock `.spacemacs` :-(

**Comment by Letheed** [comment]
Yes, I made sure everything was up-to-date when I posted the bug report – packages and spacemacs. I just removed and re-cloned the spacemacs repository and I can confirm it's still happening.

**Comment by Letheed** [comment]
Here's my config if you want to check.
[spacemacs.txt](https://github.com/syl20bnr/spacemacs/files/139575/spacemacs.txt)

**Comment by StreakyCobra** [comment]
Starting with a fresh `.emacs.d/` and your `.spacemacs` I can reproduce. But I can't understand why it is not working. You can use this in the meantime to use the `bepo` layer:

```elisp
(bepo :variables bepo-set-disabled-configurations '(flycheck-error-list))
```

This will disable the problematic code.

**Comment by StreakyCobra** [comment]
Ok I found it. A `bepo` fix was ported to `master` (482aa9f0c943031be8d9d6e7511b228c8e68e35d), but this was fixing a change that hasn't been ported to master yet (e9f8377b9252e258afecf30d24d8106bda46dd3d).

@syl20bnr This `bepo` fix should probably be reverted in the next minor release (if any), and deployed only with the next major release. But considering the number of `bepo` users it should not be the only reason of doing a minor release, especially because the workaround using the variable is elegant and simple.

**Comment by Letheed** [comment]
Great, thanks for your efforts.

**Comment by syl20bnr** [comment]
Oops sorry for this I will silently hotfix it.

**Comment by syl20bnr** [comment]
This should be fixed in 0.105.12
